recipes/arrow/all/patches/1.0.0-0001-cmake.patch

```diff
@@ -62,32 +29,17 @@
      if(NOT Snappy_FOUND)
        find_package(SnappyAlt REQUIRED)
      endif()
-   endif()
- 
-   # TODO: Don't use global includes but rather target_include_directories
--  get_target_property(SNAPPY_INCLUDE_DIRS Snappy::snappy INTERFACE_INCLUDE_DIRECTORIES)
-+  get_target_property(SNAPPY_INCLUDE_DIRS Snappy::Snappy INTERFACE_INCLUDE_DIRECTORIES)
-   include_directories(SYSTEM ${SNAPPY_INCLUDE_DIRS})
- endif()
- 
 @@ -938,7 +938,7 @@ endmacro()
  if(ARROW_WITH_BROTLI)
    resolve_dependency(Brotli)
    # TODO: Don't use global includes but rather target_include_directories
 -  get_target_property(BROTLI_INCLUDE_DIR Brotli::brotlicommon
-+  get_target_property(BROTLI_INCLUDE_DIR Brotli::Brotli
-                       INTERFACE_INCLUDE_DIRECTORIES)
-   include_directories(SYSTEM ${BROTLI_INCLUDE_DIR})
- endif()
-@@ -1057,7 +1057,7 @@ endmacro()
- if(ARROW_USE_GLOG)
-   resolve_dependency(GLOG)
-   # TODO: Don't use global includes but rather target_include_directories
--  get_target_property(GLOG_INCLUDE_DIR glog::glog INTERFACE_INCLUDE_DIRECTORIES)
-+  get_target_property(GLOG_INCLUDE_DIR GLOG::GLOG INTERFACE_INCLUDE_DIRECTORIES)
-   include_directories(SYSTEM ${GLOG_INCLUDE_DIR})
+-                      INTERFACE_INCLUDE_DIRECTORIES)
+-  include_directories(SYSTEM ${BROTLI_INCLUDE_DIR})
++  #get_target_property(BROTLI_INCLUDE_DIR Brotli::brotlicommon
++  #                   INTERFACE_INCLUDE_DIRECTORIES)
++  include_directories(SYSTEM ${CONAN_INCLUDE_DIRS_BROTLI})
```

Comment:
Doesn't it work without patch?

Reply:
I mean: if `get_target_property(GLOG_INCLUDE_DIR glog::glog INTERFACE_INCLUDE_DIRECTORIES)` seems to work (because it was restored) why not `get_target_property(BROTLI_INCLUDE_DIR Brotli::brotlicommon INTERFACE_INCLUDE_DIRECTORIES)`?

Reply:
Indeed, works on Linux.
Let's test it on ci now.